### PR TITLE
API documentation for staged requests

### PR DESCRIPTION
**Pull request by bgeuken**

### CODE REVIEW

**Comment by hennevogel** [comment]
If you add an example, without a schema file it needs to be added in the [exception list](https://github.com/openSUSE/open-build-service/blob/master/src/api/test/functional/backend_test.rb#L14) in the test that is failing...

File: docs/api/api/api.txt [hunk 1]
```diff
@@ -1944,6 +1943,21 @@ POST /staging_excluded_requests/<request>/<project>
...
 
+=== Staged Requests
+
+<staging_project>: The name of the staging project, e.g. home:user1:Staging:A
+
+GET /staging_projects/<staging_project>/staged_requests
+  Get all staged requests of a staging project.
+XmlResult: staged_requests
+
+POST /staging_projects/<staging_project>/staged_requests
```
Comment (coolo) [inline docs/api/api/api.txt]: what is missing here is the format expected. It's something with <number>1</number> from what I saw

Comment (vpereira) [reply docs/api/api/api.txt]: after the last changes, the routes look more like:

```
/staging/:staging_main_project_name/staging_projects/:staging_project_name/staged_requests(.:format)
```

**Comment by bgeuken** [comment]
@vpereira Updated

**Pull request merged**